cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaMirrorMakerClusterTest.java

```diff
@@ -0,0 +1,355 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model;
+
+import io.fabric8.kubernetes.api.model.ConfigMap;
+import io.fabric8.kubernetes.api.model.Container;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.EnvVarBuilder;
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.api.model.OwnerReference;
+import io.fabric8.kubernetes.api.model.extensions.Deployment;
+import io.strimzi.api.kafka.model.CertSecretSourceBuilder;
+import io.strimzi.api.kafka.model.KafkaMirrorMaker;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerAuthenticationTlsBuilder;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerBuilder;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerConsumerSpec;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerConsumerSpecBuilder;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerProducerSpec;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerProducerSpecBuilder;
+import io.strimzi.api.kafka.model.KafkaMirrorMakerSpec;
+import io.strimzi.operator.cluster.ResourceUtils;
+import io.strimzi.operator.common.model.Labels;
+import io.strimzi.test.TestUtils;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class KafkaMirrorMakerClusterTest {
+    private final String namespace = "test";
+    private final String cluster = "mirror";
+    private final int replicas = 2;
+    private final String image = "my-image:latest";
+    private final String metricsCmJson = "{\"animal\":\"wombat\"}";
+    private final String configurationJson = "{\"foo\":\"bar\"}";
+    private final String producerBootstrapServers = "foo-kafka:9092";
+    private final String consumerBootstrapServers = "bar-kafka:9092";
```

Comment:
just a nit. Maybe we can use meaningful names just because we know they are source and target so .... `bar-....` -> `source-....` and `foo-....` -> `target-....`